howso/direct/client.py

```diff
@@ -1314,10 +1324,11 @@ def train(  # noqa: C901
                     input_is_substituted=input_is_substituted,
                     series=series,
                     session=self.active_session.id,
-                    train_weights_only=train_weights_only
+                    train_weights_only=train_weights_only,
+                    skip_auto_analyze=skip_auto_analyze,
                 )
                 if response and response.get('status') == 'analyze':
-                    auto_analyze = True
+                    needs_analyze = True
```

Comment:
Drop the "default" assignment on line 1299 and change this line to:

```
needs_analyze = response and response.get('status') == 'analyze'
```

Reply:
discussed this, keeping it the way it is.